.github/workflows/generic_test.yml

```
@@ -45,7 +48,7 @@ jobs:
           cache-from: type=local,src=/tmp/.buildx-cache
 
       - name: 'Run tests'
-        run: make generate-accounts tests
+        run: make generate-accounts test/part/${{ matrix.part }}
```

Am I right, that `make generate-accounts` is run 4 times? For each `matrix.part`? If so, lets change it to:

```suggestion
	name: Prepare tests
        run: make generate-accounts
        name: Run tests
        run: make test/part/${{ matrix.part }}
```

---

I will wait with this until after @polarathene has taken care of the history :)

---

Done, sorry for the delay!

It did not help that much with the review UI on Github (_due to a merge commit diff I think_), but the history looks like it should be preserved fine with a rebase merge now (_Github blame won't likely honour it, but other `git blame` tooling should_).